<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeTerms</class>
 <widget class="QWidget" name="ChangeTerms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(37, 41, 109);
color: rgb(255, 255, 0);</string>
  </property>
  <widget class="QPushButton" name="btnMainMenu">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.966, stop:0.85 #CAF2FA, stop:0.99 #656B87);
color: rgb(230, 225, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
qproperty-alignment: AlignCenter;
color: rgb(71, 113, 177);
border-radius: 25px;</string>
   </property>
   <property name="text">
    <string>Go Back to Main</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="title">
    <string>Read the File you want and edit the term</string>
   </property>
   <widget class="QLabel" name="lblTerms">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radEasy">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Easy</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radMedium">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radHard">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hard</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTerms">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEditTerms">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(170, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWriteFile">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;After file is read and term is changed click here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.966, stop:0.85 #CAF2FA, stop:0.99 #656B87);
color: rgb(230, 225, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
qproperty-alignment: AlignCenter;
color: rgb(71, 113, 177);
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>Change Terms</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReadFile">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Read the contents of desired difficulty&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.966, stop:0.85 #CAF2FA, stop:0.99 #656B87);
color: rgb(230, 225, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
qproperty-alignment: AlignCenter;
color: rgb(71, 113, 177);
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>Read File</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
